__tests__/items.test.js

```diff
@@ -0,0 +1,41 @@
+let item = {
+    name: 'longsword',
+    type: 'weapon',
+    durability: 100,
+    enhancement: 0
+}
+
+const newItem = {
+    durability: 100,
+    enhancement: 0
+}
+
+test('item req', () => {
+    expect({ a: 1 }).toHaveProperty('a', 1)
+    expect(a).toEqual('name')
+    //since the value of name can be anything, I don't think it needs testing.
+    expect({ b: 2 }).toHaveProperty('b', 2)
+    expect(b).toEqual('type')
+    expect(2).toEqual([expect.stringContaining('weapon'),
+        expect.stringContaining('armor')])
+    expect({ c: 3 }).toHaveProperty('c', 3)
+    expect(c).toEqual('durability')
+    expect(3).toBeLessThanOrEqual(100)
+    expect({ d: 4 }).toHaveProperty('d', 4)
+    expect(d).toEqual('enhancement')
+    expect(4).toBeGreaterThanOrEqual(0)
+    expect(4).toBeLessThanOrEqual(15)
+    expect(4).toMatch('PRI')
```

Comment:
Start at the basics, with maybe the first test you wrote with the proper test items. After you see that those are working as expected, I would move to these.